pkg/pillar/types/zedroutertypes_test.go

```diff
@@ -4,10 +4,24 @@
 package types
 
 import (
+	"github.com/satori/go.uuid"
 	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
+var underlayUUID = uuid.UUID{0x6b, 0xa7, 0xb8, 0x10, 0x9d, 0xad, 0x11, 0xd1,
+	0x80, 0xb4, 0x00, 0xc0, 0x4f, 0xd4, 0x30, 0xc8}
+var overlayUUID = uuid.UUID{0x6b, 0xa7, 0xb8, 0x10, 0x9d, 0xad, 0x11, 0xd1,
```

Comment:
Shouldn't they be different?

Reply:
Yep, I must have copied the wrong thing. Let me fix that.

Reply:
I'm confused. Wasn't this part supposed to be fixed @archishou ?

Reply:
I did: https://github.com/lf-edge/eve/pull/108/commits/8c9cd9a065f6a859db98677646207d0642e0eb6c